### PR TITLE
Feature/navigate printbugger to probe

### PR DESCRIPTION
**Pull request by LinaUr**
Adds a halt button to the lines of the printbugger which can be used to navigate to the exact position of the probes value in the trace.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -27,17 +27,17 @@ jobs:
         env: 
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - run: ./scripts/build_image.sh
-        if: matrix.os == 'ubuntu-latest' && (github.ref == 'refs/heads/main')
+        if: matrix.os == 'ubuntu-latest' && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop')
```
Comment (codeZeilen) [inline .github/workflows/main.yml:30]: There is something funky happening here. The new version is the wrong version :)

File: packages/Babylonian-Core.package/BPTracer.class/instance/checkForHaltAt..st [hunk 2]
```diff
@@ -1,4 +1,7 @@
 private
 checkForHaltAt: aProbeId
 	
-	haltProbeId == aProbeId ifTrue: [BPExampleExecutionInterrupted signal].
\ No newline at end of file
+	haltProbeId == aProbeId ifTrue: [
+		haltProbeIdCount == probeIdCount
+			ifTrue: [	BPExampleExecutionInterrupted signal]
```
Comment (codeZeilen) [inline packages/Babylonian-Core.package/BPTracer.class/instance/checkForHaltAt..st:6]: There is a funny tab here :)

File: packages/Babylonian-Printbugger.package/BPSwimlane.class/instance/haltButtonClickedFor.and..st [hunk 3]
```diff
@@ -0,0 +1,10 @@
+private
+haltButtonClickedFor: aTraceValue and: aBPExample
+	
+	| probe positionInProbe |
+	probe := (aBPExample trace trace probes) at: (aTraceValue belongingProbeId).
```
Comment (codeZeilen) [inline packages/Babylonian-Printbugger.package/BPSwimlane.class/instance/haltButtonClickedFor.and..st:5]: Die Klammern sind hier zuviel :)

File: packages/Babylonian-Printbugger.package/BPSwimlane.class/instance/newHaltButtonFor..st [hunk 4]
```diff
@@ -0,0 +1,13 @@
+private
+newHaltButtonFor: aTraceValue
+	
+	| exploreButton |	
+	
+	exploreButton := BPIconicButton newHaltButtonOfSize: (self iconSize) andExtentOffsetX: self scrollBarSize.
```
Comment (codeZeilen) [inline packages/Babylonian-Printbugger.package/BPSwimlane.class/instance/newHaltButtonFor..st:6]: Die Klammern #iconSize können weg :)

File: scripts/prepare_image.st [hunk 5]
```diff
@@ -19,7 +19,7 @@ FileStream fileIn: 'SOHCharacter.cs'.
 (Smalltalk at: #Metacello) new
   baseline: 'BabylonianProgramming';
   repository: 'filetree:///home/runner/work/babylonian-programming-smalltalk/babylonian-programming-smalltalk/packages';
-  load: 'full'.
+  load: 'tests'.
```
Comment (codeZeilen) [inline scripts/prepare_image.st:22]: This is also the wrong version. It should be 'full'

Comment (LinaUr) [reply scripts/prepare_image.st:22]: how can I fix this? I tried merging dev and liveprog into this branch yesterday.

Comment (codeZeilen) [reply scripts/prepare_image.st:22]: As the merge already happened, you can simply change it back and commit it as a correction :)

**Pull request merged**